app/javascript/widget/App.vue

```diff
@@ -100,10 +100,8 @@ export default {
     },
     registerUnreadEvents() {
       bus.$on('on-agent-message-recieved', () => {
-        if (!this.isIFrame) {
```

Comment:
isIFrame will be true for us so `this.setUserLastSeen();` was not running

Reply:
Normally, chatwoot would set the user-last-seen-at when you open the chat from the "See new messages" button on the unread view. So if you receive a message, then refresh, the unread view will come up because user-last-seen-at wouldn't have been set. Which doesn't really _seem_ correct, but that's what it was doing. Since we got rid of the unread view user-last-seen-at wasn't ever being updated so the chat window would keep popping up for "unseen" messages